test/bridge/testpyscf.py

```diff
@@ -8,35 +8,39 @@
 import unittest
 
 import numpy as np
-
+from ..test_data import getdatafile
```

Comment:
Is it possible to do this without a relative import?

Reply:
Hm, I will try! The Horton test is currently also using a relative import which is what I modeled this after, but this is likely possible without it.

Reply:
The import of the test_data module has been added to the init file of the test directory. With this in place, the `getdatafile` function without a relative path. Is this an acceptable solution?